pkg/asset/internal/templates.go

```diff
@@ -19,7 +19,7 @@ contexts:
     user: kubelet
 `)
 
-var KubeSystemSARoleBindingTemplate = []byte(`apiVersion: rbac.authorization.k8s.io/v1alpha1
+var KubeSystemSARoleBindingTemplate = []byte(`apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Should be `v1` per the release notes draft: https://github.com/kubernetes/features/blob/master/release-1.8/release_notes_draft.md#auth-1

Reply:
Either works. v1alpha1 doesn't anymore due to https://github.com/kubernetes/kubernetes/pull/51839